### PR TITLE
Stats content card component

### PR DESCRIPTION
**Pull request by xifengjin88**
I tested this feature locally and requesting for review!!

### CODE REVIEW

File: browser/components/StatsContentCard/StatsContentCard.scss [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+  width: 30%;
+  height: 400px;
+  background-image:
+    linear-gradient(
+      rgba(0, 0, 0, 0.5),
+      rgba(0, 0, 0, 0.5)
+    ),
+    url('http://localhost:5000/images/sun_wu_kong.jpeg');
+  background-size: cover;
+  &-label {
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: add a space here between these extensions

File: browser/components/StatsContentCard/StatsContentCard.scss [hunk 2]
```diff
@@ -0,0 +1,41 @@
+.stat-content-card {
+  display: flex;
+  box-sizing: border-box;
+  position: relative;
+  border: solid 10px #335183;
+  transition: border-width 0.6s linear;
+  width: 30%;
+  height: 400px;
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: hmm i feel like we should only be constraining one of these things.

File: browser/components/StatsContentCard/StatsContentCard.scss [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  background-image:
+    linear-gradient(
+      rgba(0, 0, 0, 0.5),
+      rgba(0, 0, 0, 0.5)
+    ),
+    url('http://localhost:5000/images/sun_wu_kong.jpeg');
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: why is this here? aren't we rendering this via a passed in link?

File: browser/components/StatsContentCard/StatsContentCard.scss (hunk 2 above)
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: can we use ems please?

File: browser/components/StatsContentCard/StatsContentCard.scss [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+    width: 100%;
+    font-size: 30px;
+    text-align: center;
+    font-family: 'Passion One', cursive;
+    background-color: rgba(0, 0, 0, 0);
+    color: #fff;
+    align-self: flex-end;
+    padding-bottom: 5%;
+    &-most-kill {
+      font-size: 3rem;
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: I'm okay with rems instead of ems just be consistent.

File: browser/components/StatsContentCard/StatsContentCard.scss [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+    ),
+    url('http://localhost:5000/images/sun_wu_kong.jpeg');
+  background-size: cover;
+  &-label {
+    width: 100%;
+    font-size: 30px;
+    text-align: center;
+    font-family: 'Passion One', cursive;
+    background-color: rgba(0, 0, 0, 0);
+    color: #fff;
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: Please capitalize. One thing to consider is to start making a colors style sheet.

```
$color-black: #FFFFFF;
```
now you can use that variable.

File: browser/components/StatsContentCard/StatsContentCard.scss [hunk 6]
```diff
@@ -0,0 +1,41 @@
...
+    padding-bottom: 5%;
+    &-most-kill {
+      font-size: 3rem;
+    }
+
+    &-god-name {
+      font-size: 4rem;
+    }
+  }
+  &-label > p {
```
Comment (coreygreenwald) [inline browser/components/StatsContentCard/StatsContentCard.scss]: are you having an issue with built in default paragraph styling? if so either a add a class or b reset the base styling of paragraph tags.

**Comment by xifengjin88** [comment]
screen shot 2018-02-02 at 1 43 35 pm
